src/main/java/com/cloudbees/hudson/plugins/folder/relocate/StandardHandler.java

```diff
@@ -88,42 +139,44 @@ public HandlingMode applicability(Item item) {
                 if (!permitted(item, itemGroup)) {
                     continue;
                 }
-                ItemGroup<?> p = itemGroup;
-                if (p instanceof Item) {
-                    Item i = (Item) p;
```

Comment:
`i == g` don't know why we were casting an `Item` to `ItemGroup` and then checking if it was still an `Item` and then casting back to `Item`... it was an `Item` all along and we had it in the variable `g` already!